packages/tentales/src/service/create-service-callers.ts

```diff
@@ -9,7 +11,18 @@ function getServiceConfig(serviceName: string, config: any): ServiceConfig {
 }
 
 export function createServiceCallers(config: Config): ServiceCaller[] {
-  const bearer = `Bearer ${mintToken(config.auth.serverSecret)}`
+  const headers = new Headers()
+  headers.append("Accept", "application/json")
+  headers.append("Content-Type", "application/json")
```

Comment:
Is `Headers` needed here? Or will just an object literal will suffice? Or maybe Headers is more explicit...

Reply:
I had some issues with the typing when I made the authorization-header optional. Using Headers this way resolved the typing issues and I figured it was good enough :)